pkg/pprof/fix_go_truncated.go

```diff
@@ -23,11 +23,20 @@ const (
 // MayHaveGoHeapTruncatedStacktraces reports whether there are
 // any chances that the profile may have truncated stack traces.
 func MayHaveGoHeapTruncatedStacktraces(p *profilev1.Profile) bool {
-	if !hasGoHeapSampleTypes(p) {
+	var minDepth int
```

Comment:
Can't make a suggestion but we should rename the function:
```go
func MayHaveGoTruncatedStacktraces(p *profilev1.Profile) bool {
```

Reply:
Maybe `PotentialTruncatedGoStacktraces`?